serverHandlers/authHandler.js

```diff
@@ -8,10 +8,10 @@ const UserModel = require('../Schemas/UserSchema');
 
 async function authenticate({email, password}) {
     const user = await UserModel.findOne({ email: email });
-
+    // console.log('here');
     if(user && bcrypt.compareSync(password, user.password)) {
         const { password, ...restInfo } = user.toObject();
-        const token = jwt.sign({sub: user._id}, config.secret);
+        const token = jwt.sign({sub: user._id}, config.secret, {expiresIn: 86400});
```

Comment:
86400 could be stored in constants as well.

Reply:
done